packages/kamino-sdk/src/Kamino.ts

```diff
@@ -275,6 +300,66 @@ export class Kamino {
     return balances;
   }
 
+  private async getStrategyBalancesRaydium(strategy: WhirlpoolStrategy) {
+    let poolState = await PoolState.fetch(this._connection, strategy.pool);
+    let positionState = await PersonalPositionState.fetch(this._connection, strategy.position);
+
+    if (!positionState) {
+      throw new Error(`Raydium position ${strategy.position.toString()} could not be found.`);
+    }
+    if (!poolState) {
+      throw new Error(`Raydium pool ${strategy.pool.toString()} could not be found.`);
+    }
+
+    const decimalsA = await getMintDecimals(this._connection, poolState.tokenMint0);
+    const decimalsB = await getMintDecimals(this._connection, poolState.tokenMint1);
+
+    const aVault = await this.getTokenAccountBalance(strategy.tokenAVault);
+    const bVault = await this.getTokenAccountBalance(strategy.tokenBVault);
+    let lowerSqrtPriceX64 = SqrtPriceMath.getSqrtPriceX64FromTick(positionState.tickLowerIndex);
+    let upperSqrtPriceX64 = SqrtPriceMath.getSqrtPriceX64FromTick(positionState.tickUpperIndex);
+
+    positionState.tickLowerIndex;
+
+    const { amountA, amountB } = LiquidityMath.getAmountsFromLiquidity(
+      poolState.sqrtPriceX64,
+      new BN(lowerSqrtPriceX64),
+      new BN(upperSqrtPriceX64),
+      positionState.liquidity,
+      true
+    );
+
+    const vaultBalances: StrategyVaultBalances = {
+      a: aVault,
+      b: bVault,
+    };
+    const prices = await this.getPrices(strategy);
+    const aAvailable = new Decimal(strategy.tokenAAmounts.toNumber());
```

Comment:
toNumber shouldnt be used as it breaks if the number gets too big (like with BONK), please use toString instead

(all 4 lines should be fixed)